lib/url-analyzer/index.js

```diff
@@ -0,0 +1,37 @@
+const common = require( '../common' );
+const Memcached = require( 'memcached' );
+
+async function analyze( url, cache_key ) {
+       let  data = {
+            url: url,
+            status: 'analyzed',
+            colors: {
+                foo: 'bar' // todo, implement logic to retrieve colors
+            }
+        }
+
+        save_data(cache_key, data);
+        common.delete_from_queue( get_job_key( url ), 0 );
+}
+
+function save_data(cache_key, data) {
+    let memcached_host = process.env.MSHOTS_MEMCACHE_HOST
+        ? process.env.MSHOTS_MEMCACHE_HOST
+        : '127.0.0.1';
+
+    let memcached = new Memcached( memcached_host + ':11211' );
```

Comment:
Should the memcached port be configurable too?

Reply:
Its a good point but since it was hardcoded in the past, for now I'll leave it like this.